posthog/api/authentication.py

```diff
@@ -86,7 +86,10 @@ def create(self, validated_data: Dict[str, str]) -> Any:
         )
 
         if not user:
-            raise serializers.ValidationError("Invalid email or password.", code="invalid_credentials")
+            raise serializers.ValidationError(
+                "Invalid email or password. Make sure you have selected the right region.",
```

Comment:
Personally I don't think we should do this in the backend. I think this makes more sense that the frontend intercepts the standard error and displays an additional Alert indicating that we now have multiple regions, which one they are currently trying to login to and an easy link to take them to the proper place.

Reply:
Mmm makes sense!

Reply:
Also we should include that same special banner in other places like "forgot password"